### ISSUE TITLE
AlwaysBeRunning.net claim API not working

### ISSUE THREAD

**Issue by madarasz** [Null-Signal-Games/netrunnerdb#228]
We used to have this NetrunnerDB endpoint in collaboration with AlwaysBeRunning.net. The user could make their tournament claim appear on their NetrunnerDB decklist page.

I'm keep getting 404 errors from the API that used to work a couple of months ago. I used the NetrunnerDB API explorer to replicate the issue (https://netrunnerdb.com/oauth/explorer). 

I'm sending POST request to https://netrunnerdb.com/decklists/50385/claims with the body:

> {
>     name: "Bratislava Store Champs",
>     url: "https://alwaysberunning.net/tournaments/1217/bratislava-store-champs-march-2018",
>     rank: 13
> }

I'm getting a 404 response for my request.

**Comment by x3N1GM4x** [comment]
Do you see output like this in your browser console when this happens?
![image](https://user-images.githubusercontent.com/10621139/49652878-3b495700-fa2b-11e8-80d1-4e0e78f7c747.png)

**Comment by madarasz** [comment]
@x3N1GM4x The call is made by the backend, not the frontend. I'm only seeing it returns 404 http error to my server.

**Comment by x3N1GM4x** [comment]
Ok, I thought you said you were using the API explorer to replicate the issue, which is what I did to get the output above, just trying to also reproduce the issue.

**Comment by madarasz** [comment]
Yeah, since I don't want to give out access to my server, the API explorer seems a good way to replicate issues. The API explorer might have some problems of its own, I don't know why that is happening, hopefully it's unrelated to my issue.
